src/entities/Item.ts

```diff
@@ -21,6 +23,10 @@ class Item {
   @Column({ default: false })
   public complete?: boolean;
 
+  @Field(() => User)
+  @ManyToOne(() => User, user => user.items)
+  public user!: User;
```

Comment:
I currently think it's okay to remove `!` since the ts shows `[ts] Property 'name' has no initializer and is not definitely assigned in the constructor. [2564]`. What do you think?

Reply:
Do you mean add `strictPropertyInitialization : false` in our tsconfig to disable the rule?

Reply:
Yes.